content/browser/renderer_host/render_widget_host_view_child_frame.cc

```diff
@@ -185,8 +185,11 @@ void RenderWidgetHostViewChildFrame::SetFrameConnectorDelegate(
 #if defined(USE_AURA)
 void RenderWidgetHostViewChildFrame::SetFrameSinkId(
     const viz::FrameSinkId& frame_sink_id) {
-  if (switches::IsMusHostingViz())
+  if (switches::IsMusHostingViz()) {
     frame_sink_id_ = frame_sink_id;
+    CHECK(frame_connector_);
```

Comment:
I would relax this to a DCHECK, although CHECK isnt strictly wrong.

Reply:
done